examples/xo_rust/Cargo.toml

```diff
@@ -15,9 +15,14 @@
 
 [package]
 name = "sawtooth-xo"
-version = "0.8.0"
+version = "0.3.1"
 authors = ["Bitwise IO, Inc."]
 edition = "2018"
+license = "Apache-2.0"
+description = """\
+    Sawtooth XO is a Sawtooth Transaction Handler for executing Tic-tac-toe \
```

Comment:
Xo not XO

Reply:
It's "XO" according to the transaction family spec: https://sawtooth.hyperledger.org/docs/core/nightly/master/transaction_family_specifications/xo_transaction_family.html

Reply:
No initial caps for:
transaction handler 
tic-tac-toe 
noughts and crosses  (note the spelling correction)

All caps for:
XO